docs/how/delete-metadata.md

```diff
@@ -2,6 +2,18 @@
 
 There are a two ways to delete data from DataHub.
 
+
+## Configuring DataHub CLI
+
+The CLI will point to localhost DataHub by default. Running
+
+```aidl
```

Comment:
Why is aidl used for syntax here?

Reply:
Ah good question- intellij used this as the default and I didn't remove it

Reply:
updated!